libs/dh/wholesale/feature-settlement-reports/src/lib/tabs/dh-wholesale-settlements-reports-tabs-balance.component.html

```diff
@@ -52,13 +51,60 @@ <h3>
           "
         ></watt-dropdown>
       </watt-form-field>
-      <watt-button
-        [disabled]="!searchForm.valid"
-        *dhPermissionRequired="['settlements:manage']"
-        variant="secondary"
-        (click)="downloadClicked()"
-        >{{ t("download") }}</watt-button
-      >
     </form>
+    <watt-table
+      [dataSource]="dataSource"
+      [columns]="columns"
+      [resolveHeader]="translateHeader"
```

Comment:
suggestion: the `resolveHeader` is deprecated in favor of `header` input on the `WattTableCellDirective` directive. `dh-users-tab-table.component.html` has an example of how to use it.

Reply:
I'll fix that.

Reply:
Fixed